packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -0,0 +1,98 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Button: ComponentStyleConfig = {
+  baseStyle: {
+    fontWeight: 'medium',
+    borderRadius: 'base', // <-- border radius is same for all variants and sizes
+    color: 'text-light',
```

Comment:
this has to use the corresponding color defined in our theme

Reply:
it is using `text-light` color defined in the theme.

Reply:
you're totally right!